include/bg_window.h

```diff
@@ -87,6 +87,36 @@ enum GFBppMode {
     GF_BG_CLR_8BPP,
 };
 
+enum GFPalLoadLocation {
+    GF_PAL_LOCATION_MAIN_BG = 0,
+    GF_PAL_LOCATION_MAIN_OBJ,
+    GF_PAL_LOCATION_MAIN_BGEXT,
+    GF_PAL_LOCATION_MAIN_OBJEXT,
+    GF_PAL_LOCATION_SUB_BG,
+    GF_PAL_LOCATION_SUB_OBJ,
+    GF_PAL_LOCATION_SUB_BGEXT,
+    GF_PAL_LOCATION_SUB_OBJEXT,
+};
+
+enum GFPalSlotOffset {
```

Comment:
you mentioned that enums are giving you issues, not sure which ones are but you might be better off setting those to `#define`s instead of enums

Reply:
this would especially apply here when you're defining each offset individually anyways

Reply:
the ones giving issues are GFBgLayer, you can still pass in the values but there's no way to ensure that a valid value is getting passed in. as for the offsets, I could make them defines if you think that's better

Reply:
Honestly just make GFBgLayer defines, I remember having trouble with them when we were adding the -pedantic flag, I doubt GF implemented them well and us trying to fix it is just gonna cause a ton of issues. As long as the offsets aren't causing other issues they're fine being they way they are currently.